android/src/main/java/com/swmansion/gesturehandler/core/GestureHandlerOrchestrator.kt

```diff
@@ -480,8 +488,11 @@ class GestureHandlerOrchestrator(
             continue
           }
 
-          // we don't want to extract gestures other than hover when processing hover events
-          if (event.action in listOf(MotionEvent.ACTION_HOVER_EXIT, MotionEvent.ACTION_HOVER_ENTER, MotionEvent.ACTION_HOVER_MOVE) && handler !is HoverGestureHandler) {
+          // We don't want to extract gestures other than hover when processing hover events.
+          // There's only one exception - RootViewGestureHandler. TalkBack uses hover events,
+          // so we need to pass them into RootViewGestureHandler, otherwise press and hold
+          // gesture stops working correctly (see https://github.com/software-mansion/react-native-gesture-handler/issues/3407)
```

Comment:
I have mixed feelings on this - on the one hand, it's nice having the explanation on the call site, but on the other, it would be nice to have in alongside the implementation.

Reply:
I've moved this comment above the implementation ([c1a9158](https://github.com/software-mansion/react-native-gesture-handler/pull/3462/commits/c1a91588266e9604bcc22ccd3f5b220383cf51a2)). I think name of this function is enough self-explanatory, so we don't have to write a comment alongside call.